crates/sui-framework/sources/tx_context.move

```diff
@@ -51,8 +49,10 @@ module sui::tx_context {
        self.epoch_timestamp_ms
     }
 
-    /// Generate a new, globally unique object ID with version 0
-    public(friend) fun new_object(ctx: &mut TxContext): address {
+    /// Create an `address` that has not been used. As it is an object address, it should never
+    /// occurr as the address for a user.
+    /// In other words, the generated address is a globally unique object ID.
+    public fun fresh_object_address(ctx: &mut TxContext): address {
```

Comment:
I originally had this as `unused_object_address` but I thought maybe `fresh` conveyed this better?